<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestion_oeuvre</class>
 <widget class="QMainWindow" name="gestion_oeuvre">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1246</width>
    <height>715</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pb_pdf">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>290</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton

{

border: 2px solid black;

border-radius: 10px;

padding: 0 8px;

background: pink;

}
QPushButton:hover
{
   
	background-color: rgb(255, 85, 0);
    border: 1px solid yellow;
}

QPushButton:pressed
{
    background-color: rgb(255, 180, 5);
    border: 1px solid white;
}</string>
    </property>
    <property name="text">
     <string>Format_PDF</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>exposition date</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/logo/logo.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_entered_date">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>310</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {

border: 2px solid black;

border-radius: 10px;

padding: 0 8px;

background: white;

font-weight: bold; /* Le texte est en gras */

}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>entered_date</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_modifier">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>590</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton

{

border: 2px solid black;

border-radius: 10px;

padding: 0 8px;

background: pink;

}
QPushButton:hover
{
   
	background-color: rgb(255, 85, 0);
    border: 1px solid yellow;
}

QPushButton:pressed
{
    background-color: rgb(255, 180, 5);
    border: 1px solid white;
}
</string>
    </property>
    <property name="text">
     <string>update</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_ajouter">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>590</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton

{

border: 2px solid black;

border-radius: 10px;

padding: 0 8px;

background: pink;

}
QPushButton:hover
{
   
	background-color: rgb(255, 85, 0);
    border: 1px solid yellow;
}

QPushButton:pressed
{
    background-color: rgb(255, 180, 5);
    border: 1px solid white;
}</string>
    </property>
    <property name="text">
     <string>add</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_exposition_date">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>360</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {

border: 2px solid black;

border-radius: 10px;

padding: 0 8px;

background: white;

font-weight: bold; /* Le texte est en gras */

}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>530</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Rating</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_slot">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>570</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton

{

border: 2px solid black;

border-radius: 10px;

padding: 0 8px;

background: pink;



}
QPushButton:hover
{
   
	background-color: rgb(255, 85, 0);
    border: 1px solid yellow;
}

QPushButton:pressed
{
    background-color: rgb(255, 180, 5);
    border: 1px solid white;
}</string>
    </property>
    <property name="text">
     <string>Slot</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>type</string>
    </property>
   </widget>
   <widget class="QTableView" name="tab_oeuvre">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>70</y>
      <width>761</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_supprimer">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>300</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton

{

border: 2px solid black;

border-radius: 10px;

padding: 0 8px;

background: pink;

}
QPushButton:hover
{
   
	background-color: rgb(255, 85, 0);
    border: 1px solid yellow;
}

QPushButton:pressed
{
    background-color: rgb(255, 180, 5);
    border: 1px solid white;
}
</string>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_price">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>480</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {

border: 2px solid black;

border-radius: 10px;

padding: 0 8px;

background: white;

font-weight: bold; /* Le texte est en gras */

}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>1261</width>
      <height>651</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/back/back);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_state">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>290</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton

{

border: 2px solid black;

border-radius: 10px;

padding: 0 8px;

background: pink;

}
QPushButton:hover
{
   
	background-color: rgb(255, 85, 0);
    border: 1px solid yellow;
}

QPushButton:pressed
{
    background-color: rgb(255, 180, 5);
    border: 1px solid white;
}</string>
    </property>
    <property name="text">
     <string>statisitc</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>480</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>price</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>580</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton

{

border: 2px solid black;

border-radius: 10px;

padding: 0 8px;

background: pink;

}
QPushButton:hover
{
   
	background-color: rgb(255, 85, 0);
    border: 1px solid yellow;
}

QPushButton:pressed
{
    background-color: rgb(255, 180, 5);
    border: 1px solid white;
}</string>
    </property>
    <property name="text">
     <string>QR_CODE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_search">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>300</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton

{

border: 2px solid black;

border-radius: 10px;

padding: 0 8px;

background: pink;

}
QPushButton:hover
{
   
	background-color: rgb(255, 85, 0);
    border: 1px solid yellow;
}

QPushButton:pressed
{
    background-color: rgb(255, 180, 5);
    border: 1px solid white;
}</string>
    </property>
    <property name="text">
     <string>search</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_id">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>240</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {

border: 2px solid black;

border-radius: 10px;

padding: 0 8px;

background: white;

font-weight: bold; /* Le texte est en gras */

}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_type">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>420</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {

border: 2px solid black;

border-radius: 10px;

padding: 0 8px;

background: white;

font-weight: bold; /* Le texte est en gras */

}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_rating">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>520</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {

border: 2px solid black;

border-radius: 10px;

padding: 0 8px;

background: white;

font-weight: bold; /* Le texte est en gras */

}</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>350</y>
      <width>761</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>1381</width>
      <height>771</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/back/backprj.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>301</width>
      <height>701</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/back/421525424_1539931383656472_8078008905184343449_n.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/logo/logo++.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_16</zorder>
   <zorder>label</zorder>
   <zorder>label_18</zorder>
   <zorder>pb_pdf</zorder>
   <zorder>label_6</zorder>
   <zorder>label_3</zorder>
   <zorder>le_entered_date</zorder>
   <zorder>label_5</zorder>
   <zorder>pb_modifier</zorder>
   <zorder>pb_ajouter</zorder>
   <zorder>le_exposition_date</zorder>
   <zorder>label_9</zorder>
   <zorder>label_4</zorder>
   <zorder>pb_slot</zorder>
   <zorder>label_7</zorder>
   <zorder>tab_oeuvre</zorder>
   <zorder>pb_supprimer</zorder>
   <zorder>le_price</zorder>
   <zorder>pb_state</zorder>
   <zorder>label_8</zorder>
   <zorder>pushButton_8</zorder>
   <zorder>pb_search</zorder>
   <zorder>le_id</zorder>
   <zorder>le_type</zorder>
   <zorder>le_rating</zorder>
   <zorder>tableView_2</zorder>
   <zorder>label_17</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1246</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
